dagshub/data_engine/model/datasource.py

```diff
@@ -370,7 +370,16 @@ def as_of(self, time: Union[float, datetime.datetime]) -> "Datasource":
 
     def with_time_zone(self, tz_val: str) -> "Datasource":
         """
-        A time zone offset string in the form of "+HH:mm" or "-HH:mm"
+        A time zone offset string in the form of "+HH:mm" or "-HH:mm".
+
+        A metadata of type datetime is always stored in DB as a UTC time, when a query is done on this field
+        there are 3 options:
+
+        - Metadata was saved with a timezone, in which case it will be used.
+
+        - Metadata was saved without a timezone, in which case UTC will be used.
+
+        - with_time_zone specified a time zone and it will override whatever is in the database.
         """
```

Comment:
Can you put the link to the `with_time_zone` function on this? I had to dig into the source code to find where to specify that